yarn-project/boxes/blank-react/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM 278380418400.dkr.ecr.eu-west-2.amazonaws.com/circuits.js AS circuits
+FROM 278380418400.dkr.ecr.eu-west-2.amazonaws.com/yarn-project-base AS builder
+
+COPY . .
+
+COPY --from=circuits /usr/src/yarn-project/circuits.js/dest /usr/src/yarn-project/circuits.js/dest
```

Comment:
note there's two `FROM`, normally it would just be builder, but this one requires the circuits.js subpackage for a wasm file for the webpack build